hw1/02-palindrome.js

```diff
@@ -1,3 +1,36 @@
-const elem = document.querySelector('input');
+document.addEventListener('DOMContentLoaded', () => {
+    console.log('DOMContentLoaded event fired'); // Debugging
 
-elem.addEventListener('input', handleInput);
+    const numberInput = document.getElementById('numberInput');
+    const checkPalindromeBtn = document.getElementById('checkPalindromeBtn');
+    const resultParagraph = document.getElementById('result');
+
+    checkPalindromeBtn.addEventListener('click', () => {
+
+        const inputValue = parseInt(numberInput.value, 10);
```

Comment:
https://stackoverflow.com/questions/64672461/difference-between-event-value-and-event-target-value

If you end up refactoring your code with the changes of only using the input type, I found this discussion helpful on handling the event input.